From ed1160fad81a9a0d98e8ae7a896dbfc9abf91293 Mon Sep 17 00:00:00 2001
From: Dan Nica <dan.nica@nxp.com>
Date: Thu, 21 Jul 2022 08:13:38 +0300
Subject: [PATCH 13/42] s32: mmc: Set DTOCV to 1101b, the recommended value

According to the Reference Manual, 1101b is the value "recommended to use
for other speed mode except HS400/HS200/SDR104".

Issue: ALB-9078
Upstream-Status: Pending 

Signed-off-by: Dan Nica <dan.nica@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/nxp/s32/mmc/s32_mmc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/nxp/s32/mmc/s32_mmc.c b/drivers/nxp/s32/mmc/s32_mmc.c
index 45c8964ef..fd1fbe838 100644
--- a/drivers/nxp/s32/mmc/s32_mmc.c
+++ b/drivers/nxp/s32/mmc/s32_mmc.c
@@ -187,7 +187,7 @@ static void s32_mmc_init(void)
 
 	regdata = mmio_read_32(USDHC_SYS_CTRL);
 	regdata &= ~SYS_CTRL_DTOCV_MASK;
-	regdata |= SYS_CTRL_DTOCV(0xe);
+	regdata |= SYS_CTRL_DTOCV(0xd);
 	mmio_write_32(USDHC_SYS_CTRL, regdata);
 }
 
-- 
2.17.1

